src/infrastructure/validation/logger/LoggerManager.java

```diff
@@ -3,20 +3,56 @@
 /**
  * LoggerManager class that will be referenced by other modules,
  * for logging via the LoggerFactory
- * Implements ILogger interface and composes File and Console Loggers
+ * Implements ILogger interface and composes File and Console Loggers. 
+ * 
+ * Instantiation of FileLogger and ConsoleLogger indicated by,
+ * allowFileLogging and allowConsoleLogging,
+ * with defaults false, false.
  * 
  * @author Navaneeth M Nambiar
  */
 public class LoggerManager implements ILogger {
 
+	/** object that holds an instance of FileLogger */
+	private ILogger fileLogger;
+
+	/** object that holds an instance of ConsoleLogger */
+	private ILogger consoleLogger;
```

Comment:
These probably should be static too for single instance

Reply:
both synchronisation and static property is done for the loggerManager. Should I keep it for console and file loggers too?

Reply:
will have to fix this at #18  if we are changing